### PR TITLE
LON-986

### PR DESCRIPTION
**Pull request by dfernandezramos**

### CODE REVIEW

File: VAS.Drawing/CanvasObjects/Dashboard/DashboardButtonView.cs [hunk 1]
```diff
@@ -244,6 +244,14 @@ public override Selection GetSelection (Point p, double precision, bool inMotion
 
 		protected virtual void HandlePropertyChanged (object sender, PropertyChangedEventArgs e)
 		{
+			if (sender == ButtonVM && (
+				e.PropertyName == "BackgroundColor" ||
+				e.PropertyName == "TextColor" ||
+				e.PropertyName == "Name" ||
+				e.PropertyName == "TimerTime" ||
+				e.PropertyName == "Hotkey")) {
```
Comment (vguzmanp) [inline VAS.Drawing/CanvasObjects/Dashboard/DashboardButtonView.cs]: Use `nameof` for all of these.

File: VAS.Drawing/CanvasObjects/Dashboard/AnalysisEventButtonView.cs [hunk 2]
```diff
@@ -100,7 +101,13 @@ protected override void DisposeManagedResources ()
 				return ButtonVM as AnalysisEventButtonVM;
 			}
 			set {
+				if (ButtonVM != null) {
```
Comment (ylatuya) [inline VAS.Drawing/CanvasObjects/Dashboard/AnalysisEventButtonView.cs]: I think the base class is already handling this.

File: VAS.Drawing/CanvasObjects/Dashboard/DashboardButtonView.cs (hunk 1 above)
Comment (ylatuya) [inline VAS.Drawing/CanvasObjects/Dashboard/DashboardButtonView.cs]: And NeedsSync

File: VAS.Drawing/CanvasObjects/Dashboard/DashboardButtonView.cs [hunk 3]
```diff
@@ -245,11 +245,10 @@ public override Selection GetSelection (Point p, double precision, bool inMotion
 		protected virtual void HandlePropertyChanged (object sender, PropertyChangedEventArgs e)
 		{
 			if (sender == ButtonVM && (
-				e.PropertyName == "BackgroundColor" ||
-				e.PropertyName == "TextColor" ||
-				e.PropertyName == "Name" ||
-				e.PropertyName == "TimerTime" ||
```
Comment (vguzmanp) [inline VAS.Drawing/CanvasObjects/Dashboard/DashboardButtonView.cs]: Is this intentional?

**Pull request merged**